### PR TITLE
Fix parsing errors, update to be compatible with new graphml format

### PR DESCRIPTION
**Pull request by ShivanKaul**
- Remove dead code
- Remove ScriptClassic
- request type => resource type

### CODE REVIEW

File: pagegraph/src/types.rs [hunk 1]
```diff
@@ -75,6 +75,7 @@ impl From<&str> for RequestType {
         match v {
             "Image" => Self::Image,
             "ScriptClassic" => Self::ScriptClassic,
+            "Script" => Self::ScriptClassic,
```
Comment (ShivanKaul) [inline pagegraph/src/types.rs:78]: I'm not sure why we have ScriptClassic in the first place.

Comment (antonok-edm) [reply pagegraph/src/types.rs:78]: Should we rename `ScriptClassic` to `Script` then? We control 100% of the format, I don't think it makes sense to keep both around if we're gonna alias them like this here.

File: pagegraph/src/from_xml.rs [hunk 2]
```diff
@@ -731,10 +731,12 @@ impl KeyedAttrs for types::EdgeType {
                 headers: drain_string!("headers"),
                 size: drain_string!("size"),
             },
-            "request start" => Self::RequestStart {
-                request_type: crate::types::RequestType::from(&drain_string!("request type")[..]),
+            "request start" => {
+                Self::RequestStart {
+                request_type: crate::types::RequestType::from(&drain_string!("resource type")[..]),
```
Comment (antonok-edm) [inline pagegraph/src/from_xml.rs:736]: indentation looks a little off here :thinking:

File: pagegraph/src/types.rs (hunk 1 above)
Comment (ShivanKaul) [reply pagegraph/src/types.rs:78]: Ya that makes sense

Comment (ShivanKaul) [reply pagegraph/src/types.rs]: I wasn't sure why it existed in the first place

File: pagegraph/src/from_xml.rs (hunk 2 above)
Comment (ShivanKaul) [reply pagegraph/src/from_xml.rs]: Fixed

File: pagegraph/src/types.rs (hunk 1 above)
Comment (ShivanKaul) [reply pagegraph/src/types.rs]: I mean, because that's what the graphml file looked like I'm sure. But that seems to have changed.

**Review by antonok-edm — APPROVED** [review]

**Pull request merged**